packages/lesswrong/components/review/PostsItemReviewVote.tsx

```diff
@@ -26,10 +38,13 @@ const styles = (theme: ThemeType): JssStyles => ({
     paddingBottom: 4,
     ...theme.typography.body2,
     color: theme.palette.primary.main
+  },
+  marginRight: {
+    marginRight: 10
   }
 })
 
-const PostsItemReviewVote = ({classes, post}: {classes:ClassesType, post:PostsListBase}) => {
+const PostsItemReviewVote = ({classes, post, marginRight=true}: {classes:ClassesType, post:PostsListBase, marginRight?: boolean}) => {
```

Comment:
I don't think this field needs to be optional/nullable if you've provided a default argument. However, I've been having issues with default arguments actually working.